barretenberg/cpp/src/barretenberg/dsl/acir_proofs/honk_contract.hpp

```diff
@@ -782,26 +735,26 @@ library RelationsLib {
 
     function accumulatePermutationRelation(
         Fr[NUMBER_OF_ENTITIES] memory p,
-        Transcript memory tp,
+        Honk.RelationParameters memory rp,
```

Comment:
This two letter name is better than the single letter above it but cant it at least be params or something haha

Reply:
will address in followup :)